astropy/io/fits/connect.py

```diff
@@ -89,6 +89,11 @@ def read_table_fits(input, hdu=None, astropy_native=False):
     astropy_native : bool, optional
         Read in FITS columns as native astropy objects where possible instead
         of standard Table Column objects. Default is False.
+    memmap : bool, optional
+        Whether to use memory mapping, which accesses data on disk as needed. If
+        you are only accessing part of the data, this is often more efficient.
+        If you want to access all the values in the table, and you are able to
+        fit the table in memory, you may want to try turning memory mapping off.
```

Comment:
If the default is `False`, then this should probably read "you are probably better off leaving memory mapping off." (or something similar).

Reply:
Done (will push changes once everything addressed)